skins/cat17/package.json

```diff
@@ -30,6 +30,7 @@
   "devDependencies": {
     "browser-sync": "^2.18.13",
     "gulp": "^3.9.1",
+    "gulp-cli": "^1.4.0",
```

Comment:
This surprises me as I did never install that as a direct dependency, assume it was installed indirectly.